pkg/quota/controller/clusterquotamapping/clusterquotamapping.go

```diff
@@ -501,8 +501,8 @@ func (c *ClusterQuotaMappingController) waitForSyncedStores(ready chan<- struct{
 }
 
 func (c *ClusterQuotaMappingController) deleteNamespace(obj interface{}) {
-	ns, ok := obj.(*kapi.Namespace)
-	if !ok {
+	ns, ok1 := obj.(*kapi.Namespace)
+	if !ok1 {
```

Comment:
Other than doing a 
```go
var ok bool
```

or having separate names (`namespaceOK`, `tombstoneOK`, *etc*), I don't really see anything that can be better here.